src/components/apps/launch/validate.js

```diff
@@ -38,8 +37,8 @@ const validateText = ({ value, validators }) => {
                         try {
                             const regex = new RegExp(regexPattern);
                             if (!regex.test(value)) {
-                                validatorMsg = getMessage("validationRegex", {
-                                    values: { regexPattern },
+                                validatorMsg = t("validationRegex", {
+                                    values: regexPattern,
```

Comment:
I don't think the `values` key is required anymore; note that the pattern is not displaying in the validation message in Chromatic's build of the `Text Params` story when all numbers are used for the "Single-line Text" field: https://www.chromatic.com/component?appId=5ebad5c6b991d60022b1e72e&name=Apps%20%2F%20Launch&buildNumber=582&specName=Text%20Params
```suggestion
                                    regexPattern,
```